{"uuid": "7421929a-7359-468c-8895-960cb0afcd1e", "children": ["3ec9b7b2-425e-4d05-9afe-d94ff0902502"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: cannot parse capability: browserVersion\nfrom invalid argument: cannot be empty\nStacktrace:\n#0 0x55d6fae4a133 <unknown>\n#1 0x55d6fab7e966 <unknown>\n#2 0x55d6fab9ba51 <unknown>\n#3 0x55d6faba2551 <unknown>\n#4 0x55d6fab9b1b3 <unknown>\n#5 0x55d6fabe2025 <unknown>\n#6 0x55d6fabe1a11 <unknown>\n#7 0x55d6fabe3386 <unknown>\n#8 0x55d6fabe318f <unknown>\n#9 0x55d6fabda9a3 <unknown>\n#10 0x55d6fabaf46a <unknown>\n#11 0x55d6fabb055e <unknown>\n#12 0x55d6fae09cae <unknown>\n#13 0x55d6fae0d8fe <unknown>\n#14 0x55d6fae16f20 <unknown>\n#15 0x55d6fae0e923 <unknown>\n#16 0x55d6fade1c0e <unknown>\n#17 0x55d6fae31b08 <unknown>\n#18 0x55d6fae31c97 <unknown>\n#19 0x55d6fae42113 <unknown>\n#20 0x7fbbbfa94b43 <unknown>\n\n", "trace": "  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/andrei/Desktop/code/devops-sandbox/conftest.py\", line 26, in driver\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1684920098284, "stop": 1684920100464}], "start": 1684920098284, "stop": 1684920100506}